src/main/java/frc/robot/subsystems/swerve/GyroCanandGyro.java

```diff
@@ -0,0 +1,29 @@
+package frc.robot.subsystems.swerve;
+
+import com.reduxrobotics.sensors.canandgyro.Canandgyro;
+import edu.wpi.first.math.geometry.Pose2d;
+import frc.lib.util.swerve.SwerveModule;
+
+/** IO layer for CanandGyro */
+public class GyroCanandGyro implements SwerveIO {
```

Comment:
U need to make a new IO interface. the Gyros DO NOT implement swerveIO, they should implement GyroIO.